other/require_emptydir_requests_limits/require-emptydir-requests-limits.yaml

```diff
@@ -14,43 +14,43 @@ metadata:
       Pods which mount emptyDir volumes may be allowed to potentially overrun
       the medium backing the emptyDir volume. This sample ensures that any
       initContainers or containers mounting an emptyDir volume have
-      ephemeral-storage requests and limits set.   
+      ephemeral-storage requests and limits set.
```

Comment:
Probably update the description here to state the modification.

Reply:
done